### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by raproid**
add legacy docs

### CODE REVIEW

File: docs/Earn/liquid-crowdloan/parachain-liquid-crowdloan.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
 ---
-title: Parachain Liquid Crowdloans
-id: liquid-crowdloans
+title: Parachain Liquid Crowdloan
```
Comment (ddocs) [inline docs/Earn/liquid-crowdloan/parachain-liquid-crowdloan.md:2]: Crowdloans - add an 's'.

File: docs/Earn/liquid-staking/avax/avax-liquid-staking-faq.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+### Is there a maximum amount I can stake?
+
+No, you can stake at your discretion, unlimited.  
+
+
+### What is the unbonding period for aAVAXb?
+
+You receive the unstaked amount after the end of the current validating period. A validating period is 4 weeks.
+
+###
```
Comment (ddocs) [inline docs/Earn/liquid-staking/avax/avax-liquid-staking-faq.md:30]: Remove extra heading 3.

**Review by ddocs — APPROVED** [review]
Nice work. Just a minor thing to add an extra 's' to the Crowdloans.

File: docs/Earn/liquid-crowdloan/parachain-liquid-crowdloan.md (hunk 1 above)
Comment (raproid) [reply docs/Earn/liquid-crowdloan/parachain-liquid-crowdloan.md]: The thing is our product is called Liquid Crowdloan. And I'm aligning the docs with the product name on the site

File: docs/Earn/liquid-staking/avax/avax-liquid-staking-faq.md (hunk 2 above)
Comment (raproid) [reply docs/Earn/liquid-staking/avax/avax-liquid-staking-faq.md]: Thanks!

**Pull request merged**